### ISSUE TITLE
On-the-fly validation

### ISSUE THREAD

**Issue by martindholmes** [Conal-Tuohy/jEdit-TEIPlugin#17]
I may have missed something, but I can't find an option to turn on validation so that it will run in idle time, effectively validating as you edit. Validation works nicely when invoked through a shortcut, but it would be nice to have it running regularly in the background so encoders see their errors more or less immediately.

**Comment by jamescummings** [comment]
Yes, some sort of option for continual validation after X seconds of idle time would be excellent.

**Comment by MarjorieBurghart** [comment]
Hi! I don't understand  why you want validation running in idle time? What could possibly change in idle time? 
You can configure validation on key-stroke, and choose the delay after the keystroke, isn't that enough?

**Comment by martindholmes** [comment]
I didn't know you could configure validation on keystroke, or how to do that. I haven't managed to find the documentation. Could you point me at it?

**Comment by Conal-Tuohy** [comment]
Schema validation is not part of the TEI Plugin; it's a feature of the XML Plugin, and it's actually functionally invoked by the Sidekick Plugin (which is responsible for producing a hierarchical outline of the document you're editing). To configure the trigger for validation, you need to go to the Plugins / Plugin Settings menu, and in the dialog box look at the Sidekick plugin's "general" settings. You can see there's a section labelled "Auto parsing settings". 

I think it would be helpful to include this information in the TEI Plugin's documentation, and I think the TEI Plugin should probably also set these Sidekick options automatically, too, when it's first installed.

**Comment by martindholmes** [comment]
Ah, I would never have known, not being a JEdit user. That works perfectly.
